lighthouse-cli/test/smokehouse/dobetterweb/dbw-expectations.js

```diff
@@ -194,6 +194,20 @@ module.exports = [
       }
     }
   }, {
+    initialUrl: 'https://www.chromestatus.com/features',
```

Comment:
Removed this entry and added the 'dom-size' audit to the pwa_expectations file. Boom.